### PR TITLE
BC-4853 - refactor common components

### PR DESCRIPTION
**Pull request by christian-darsow**
# Short Description
Refactor common components like CKEditor to also follow the new block structure

## Links to Ticket and related Pull-Requests
https://ticketsystem.dbildungscloud.de/browse/BC-4853

## Checklist before merging

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team
- [x] DEV: Every new component is implemented having accessibility in mind (e.g. aria-label, role property)

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

File: src/components/feature-common-date-time-picker/index.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import DateTimePickerVue from ".DateTimePicker.vue";
+
+export { DateTimePickerVue };
```
Comment (MartinSchuhmacher) [inline src/components/feature-common-date-time-picker/index.ts:3]: why `DateTimePickerVue` and not just `DateTimePicker` ?

**Review by MartinSchuhmacher — COMMENTED** [review]
In general I'm not sure about the `common` usage and if its needed. Why not just use `feature-date-time-picker`, `feature-editor` and `feature-render-html` ? just so I understand it better

File: src/components/feature-common-date-time-picker/index.ts (hunk 1 above)
Comment (christian-darsow) [reply src/components/feature-common-date-time-picker/index.ts]: That was not on purpose... fixed that typo

**Comment by christian-darsow** [comment]
> In general I'm not sure about the `common` usage and if its needed. Why not just use `feature-date-time-picker`, `feature-editor` and `feature-render-html` ? just so I understand it better

Hm, well, like on the folder structure (before) I wanted to show, that all these three features belong somehow together and with that "common" indicate that they are somehow global and shared. In contrast  to e.g. all feature-board-* components. But as you ask now and I think again, that must really not be needed at all... and everything without something like "board" or "common" is just "global". ... I will change it and remove that common

**Review by MartinSchuhmacher — APPROVED** [review]

**Pull request merged**